.ado/publish.yml

```diff
@@ -492,13 +492,13 @@ extends:
                   configuration: Release
                 - platform: x86
                   configuration: Release
-                - platform: ARM64
+                - platform: ARM64EC
```

Comment:
I think we need to have __both__ ARM64 and ARM64EC builds. We can't replace one with the other. There's usage cases for both. Can you please update?

Reply:
> I think we need to have **both** ARM64 and ARM64EC builds. We can't replace one with the other. There's usage cases for both. Can you please update?

This change is exclusive for the Desktop DLL (`react-native-win32.dll`). See `artifactName: Desktop`
We do not need a pure `ARM64` build for this variant.

This should not affect the UWP build of RNW.